include/openthread/border_routing.h

```diff
@@ -239,6 +239,22 @@ void otBorderRoutingSetRouteInfoOptionPreference(otInstance *aInstance, otRouteP
  */
 void otBorderRoutingClearRouteInfoOptionPreference(otInstance *aInstance);
 
+/**
+ * Sets vendor-specific options to append at the end of emitted Router Advertisement (RA) messages.
+ *
+ * The content of @p aOptions is copied internally, so can be a temporary buffer.
+ *
+ * Subsequent calls to this function will overwrite previously set value.
```

Comment:
suggest: "Subsequent calls to this function  overwrite the previously set value."